books-list.html

```diff
@@ -8,6 +8,10 @@
     <link rel="stylesheet"  type="text/css" href="application.css">
   </head>
   <body>
+    <nav class="navbar">
+      <img src="images/w-logo.png" alt="wolox_logo" class="w-logo">
```

Comment:
The `alt` attributes are used for SEO support, you need to be as expresive as you can, for example, in this case instead of `wolox_logo`, you should set it to `Wolox` or `Wolox logo` without the underscore.